tests/e2e/manifest/setup.sh

```
@@ -0,0 +1,67 @@
+
+if [ -z "$NAMESPACE" ]; then
+  echo "Error: --namespace is required"
+  exit 1
+fi
+
+echo "Using manifest directory: $MANIFEST_DIR"
+echo "Using namespace: $NAMESPACE"
+
+# Function to substitute template variables and apply
+apply_template() {
+  local template_file="$1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)